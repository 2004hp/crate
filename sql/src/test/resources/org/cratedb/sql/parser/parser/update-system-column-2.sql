update t1 set t1.bla = 'blubb' where t1."_version" = 2