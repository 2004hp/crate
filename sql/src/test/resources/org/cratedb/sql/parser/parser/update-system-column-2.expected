org.cratedb.sql.parser.parser.UpdateNode@45b11398
statementType: UPDATE
resultSet: 	
	org.cratedb.sql.parser.parser.SelectNode@6c7d83e6
	isDistinct: false
	resultColumns: 		
		org.cratedb.sql.parser.parser.ResultColumnList@245ea8af

		[0]:		
		org.cratedb.sql.parser.parser.ResultColumn@42948bd0
		exposedName: bla
		name: bla
		tableName: null
		isDefaultColumn: false
		type: null
		expression: 			
			org.cratedb.sql.parser.parser.CharConstantNode@266726d6
			value: blubb
			type: CHAR(5) NOT NULL
		reference: 			
			org.cratedb.sql.parser.parser.ColumnReference@3e26bf4c
			columnName: bla
			tableName: t1
			type: null
	fromList: 		
		org.cratedb.sql.parser.parser.FromList@61d93674

		[0]:		
		org.cratedb.sql.parser.parser.FromBaseTable@7b862bdb
		tableName: t1
		updateOrDelete: UPDATE
		null
		correlation Name: null
		null
	whereClause: 		
		org.cratedb.sql.parser.parser.BinaryRelationalOperatorNode@25a97638
		operator: =
		methodName: equals
		type: null
		leftOperand: 			
			org.cratedb.sql.parser.parser.SystemColumnReference@4cedd1e2
			columnName: _version
			tableName: t1
			type: null
		rightOperand: 			
			org.cratedb.sql.parser.parser.NumericConstantNode@31153228
			value: 2
			type: INTEGER NOT NULL
targetTableName: 	
	org.cratedb.sql.parser.parser.TableName@e3d
	t1
